app/test_main.py

```diff
@@ -1,3 +1,48 @@
 from app.main import get_human_age
 
-# write your code here
+import pytest
+
+
+@pytest.mark.parametrize(
+    "cat_age,"
+    "dog_age,"
+    "human_age", [
+        pytest.param(
+            14, 14,
+            [0, 0],
+            id="test_14_years_cat_dog_should_convert_in_0_human_age"
+        ),
+        pytest.param(
+            15, 15,
+            [1, 1],
+            id="test_15_years_cat_dog_should_convert_in_1_human_age"
+        ),
```

Comment:
It seems like the test case `test_14_years_cat_dog_should_convert_in_0_human_age` and `test_15_years_cat_dog_should_convert_in_1_human_age` are testing the same scenario. Remember the DRY (Don't Repeat Yourself) principle, each test case should be unique and test a different aspect of the function.